tests/test_misc.py

```diff
@@ -276,15 +278,15 @@ def test_progn_in_frame(self, name, src, assertion):
 
         # Make sure the register function name didn't persist.
         self.assertEqual(
-            ['__builtins__', 'global_'],
-            stackframe.f_globals.keys()
+            sorted(('__builtins__', 'global_')),
```

Comment:
eh, what if I want to make add to it later? This shows that the order of the items is important, so why not ensure that they are sorted?